frontend/src/components/window/workspace/Panel.svelte

```diff
@@ -148,11 +149,16 @@
 					{/if}
 				</LayoutRow>
 			{/each}
+			{#if panelType == 'Document'}
```

Comment:
Panel is a component used in different contexts... You could instead create a prop for `dblclickEmptySpaceAction` (I'm tired, could not think of a better name) and use that prop here https://github.com/GraphiteEditor/Graphite/blob/7e8c6cc432c0991557e6421f30d8c3c4ee14cac1/frontend/src/components/window/workspace/Workspace.svelte#L145 like it is done for `clickAction`